### ISSUE TITLE
Export on a re-initialized repo raising error

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/BBHNet#205]
If you try to run the `export` project with an already existing model repository, `clean=False`, and `platform = "tensorrt_plan", an error happens during conversion where there is a mismatch between the number of outputs in the existing model config and the number of outputs in the network.

```console
ValueError: Number of config outputs 3 doesn't match number of outputs 1 in network.
```
I'm fairly confident this is happening as a result of [these lines](https://github.com/ML4GW/hermes/blob/ca644285ed78aeffb937ffd05837f8bdd949ae87/hermes/hermes.quiver/hermes/quiver/exporters/torch_tensorrt.py#L67-L76) in `hermes`, cause by the fact that the `Merge` functionality isn't behaving as I expected, and so should be a bug there, but I'll need to confirm before I file one.

**Comment by alecgunny** [comment]
Related to #204 , should be a pretty straightforward fix if my suspicions are correct: just do a `Merge` on the inputs and outputs if they're empty in the existing config, and otherwise confirm that they match.

**Comment by alecgunny** [comment]
Being fixed with ML4GW/hermes#38
